tests/lib/pkgdb.sh

```diff
@@ -558,9 +558,10 @@ pkg_dependencies_ubuntu_classic(){
         python3-gi
         python3-yaml
         upower
-        weston
+        weston        
```

Comment:
remove extra whitespaces
```suggestion
        weston
```

Reply:
udapted. Thanks for the review